jerry-core/jcontext/jcontext.h

```diff
@@ -170,7 +170,19 @@ struct jerry_context_t
  * This part is for JerryScript which uses external context.
  */
 
-#define JERRY_CONTEXT(field) (jerry_port_get_current_context ()->field)
+/**
+ * Declares the context before the usage. It must be called in the scope where the JERRY_CONTEXT macros
+ * are called (usually called at the beginning of a function which uses context).
+ */
+
+#define JERRY_DEFINE_CURRENT_CONTEXT() \
+  struct jerry_context_t  * jerry_current_external_context = jerry_port_get_current_context ()
+
+/**
+  * Get a field from the current context.
+  */
+
```

Comment:
Ditto, no newline needed here.

Reply:
Moreover, it is out of style: please write doc comments as follows:
```c
/**
 * doc comment
 */
```
(note the number of spaces (1 instead of 2) before the asterisks.)

Additionally, there is already a 'canonical' documentation for `JERRY_CONTEXT` some lines below, for the non-system-allocator case: "Provides a reference to a field in the current context.". Reuse that here.